src/components/Feedback/Feedback.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import emailjs from 'emailjs-com';
+
+const Feedback = () => {
+    const [feedback, setFeedback] = useState('')
+    const [submitted, setSubmitted] = useState(false)
+    const [email, setEmail] = useState('')
+    emailjs.init('1AkXF-PC-6FG2Ym15')
```

Comment:
if this 1AkXF-PC-6FG2Ym15 is sensitive do put all the sensitive information into a .env file instead of using it directly